src/button.js

```diff
@@ -0,0 +1,87 @@
+import React, {Component} from 'react'
+import PropTypes from 'prop-types'
+
+import {AUTOCOMPLETE_CONTEXT} from './constants'
+import {compose} from './utils'
+
+class Button extends Component {
+  static contextTypes = {
+    [AUTOCOMPLETE_CONTEXT]: PropTypes.object.isRequired,
+  }
+
+  static propTypes = {
+    disabled: PropTypes.bool,
+    onKeyDown: PropTypes.func,
+    onClick: PropTypes.func,
+  }
+
+  constructor(props, context) {
+    super(props, context)
+    this.autocomplete = this.context[AUTOCOMPLETE_CONTEXT]
+    this.handleKeyDown = compose(this.handleKeyDown, props.onKeyDown)
+  }
+
+  keyDownHandlers = {
+    ArrowDown(event) {
```

Comment:
I think it's beneficial since it allows you to interact with the list of items. Similar to how react-aria-menubutton does when it receives focus.